lib/modules/datasource/sbt-package/index.spec.ts

```diff
@@ -18,292 +18,183 @@ describe('modules/datasource/sbt-package/index', () => {
   });
 
   describe('getPkgReleases', () => {
-    beforeEach(() => {
+    it('returns null in case of errors', async () => {
       httpMock
         .scope('https://failed_repo')
         .get('/maven/org/scalatest/')
+        .reply(404)
+        .get('/maven/org.scalatest/')
```

Comment:
why not using `.twice()` 😉

Reply:
`/maven/org/scalatest/` and `/maven/org.scalatest/` are different

Reply:
🙈